### PR TITLE
Add validations to documents page

### PR DESCRIPTION
**Pull request by stephanieharrison-maximus**
### Please check if the PR fulfills these requirements:

- [X] After these changes, the app was run and still works as expected
- [] Unit tests for these changes were added (if applicable)
- [X] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
Adds validations to Documents page so all required documents must be uploaded before it'll allow a user to proceed. The page also produces a descriptive error when this happens.

### Additional Notes:
No automated tests added yet, although hopefully they can be added with the rest down the road. I did manually test the changes, though.

### CODE REVIEW

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
The spouse documents should only be required if the account holder has a spouse. You can use previous pages (PersonalInfo, Spouse) for reference on how to make validations conditional

File: jha/src/views/enrolment/DocumentsPage.vue [hunk 1]
```diff
@@ -105,7 +114,14 @@ export default {
...
   },
   validations() {
-    const validations = {};
+    const validations = {
+      ahCRADocuments: {
+        required,
+      },
+      spouseCRADocuments: {
+        required,
+      },
```
Comment (harrymaynard-maximus) [inline jha/src/views/enrolment/DocumentsPage.vue:123]: The spouse documents should only be required if they have a spouse.

ie:
```javascript
if (this.$store.state.enrolmentModule.hasSpouse === 'Y') {
  validations.spouseCRADocuments.required = required;
}
```

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

**Comment by stephanieharrison-maximus** [comment]
> The spouse documents should only be required if the account holder has a spouse. You can use previous pages (PersonalInfo, Spouse) for reference on how to make validations conditional

I'm with you, I thought this was going to be an issue so I checked for it. Turns out, the application doesn't add the  Spouse file uploader component in the first place if there's no spouse in the data. So while the validation is still set to required, it doesn't exist in the page and it doesn't flag a validation issue when the person tries to proceed.

I deliberately didn't add a conditional like that since it seemed redundant at the time, but if you think this implementation might change down the road, I can totally add that in.

**Comment by harrymaynard-maximus** [comment]
I'd prefer it if the validation was conditional for spouse. It keeps things clear that the validation function is only being run when a spouse exists.

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good.

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**